### PR TITLE
Add provision node

### PR DESCRIPTION
**Pull request by stephenlauck**
Add recipe for configuring a provisioning-node and add provisioning node to test-kitchen.

### CODE REVIEW

**Comment by stephenlauck** [comment]
I think this PR might be better as it's own cookbook to setup a provisioning-node.

File: README.md [hunk 1]
```diff
@@ -92,10 +95,11 @@ create the minimum number of instances (3):
 
 ```
 $ export KITCHEN_YAML=.kitchen.ssh.yml
-$ kitchen create chef-server
-$ kitchen create delivery-server
-$ kitchen create build-node1
+$ kitchen converge provisioning
+$ kitchen create .*server
```
Comment (afiune) [inline README.md]: I am not sure about how many instances can users support..

File: recipes/provisioning_node.rb [hunk 2]
```diff
@@ -0,0 +1,56 @@
+# configure provision node
```
Comment (afiune) [inline recipes/provisioning_node.rb]: I like this...

File: recipes/provisioning_node.rb [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+  end
+when 'rhel'
+  execute 'installing-delivery-cli' do
+    command <<-EOF
+      curl -o delivery-cli.rpm https://s3.amazonaws.com/delivery-packages/cli/delivery-cli-20150408004719-1.x86_64.rpm
+      sudo yum install delivery-cli.rpm -y
+    EOF
+    not_if('which delivery')
+  end
+end
```
Comment (afiune) [inline recipes/provisioning_node.rb]: extra line

File: recipes/provisioning_node.rb [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+# create environments directory
+directory '/home/vagrant/delivery-cluster/environments' do
+  owner 'vagrant'
+  group 'vagrant'
+  mode '0755'
+  recursive true
+  action :create
+end
+
+# install delivery-cli
```
Comment (afiune) [inline recipes/provisioning_node.rb]: why not using the `packagecloud` cookbook? :smile:

Comment (stephenlauck) [inline recipes/provisioning_node.rb]: fixed to use packagecloud.

**Comment by tduffield** [comment]
What might help me a little is to understand the driver behind adding a provisioning node here. I assume that the intention was to create a node that would manage the cluster?

**Comment by stephenlauck** [comment]
Closing in favor of https://github.com/opscode-cookbooks/delivery-cluster/pull/118

**Pull request closed without merging**